library/Fission/User/Mutation/Error.hs

```diff
@@ -15,6 +17,8 @@ data Create = FailedDigest
 
 instance Display Create where
   display FailedDigest = "Could not create password digest"
+  display AlreadyExists = "The username or email already exists in our system"
 
 instance ToServerError Create where
   toServerError FailedDigest = err500 { errBody = "Could not create password digest" }
+  toServerError AlreadyExists = err409 { errBody = "The username or email already exists in our system" }
```

Comment:
Could reuse the `Display` message for consistency:

```haskell
toServerError AlreadyExists = err409 { errBody = display AlreadyExists }
```

Reply:
Oh I am so glad you had this recommendation. It was irking me that we had duplicated messages!

Reply:
Happy to help 🙌

Something that may be of interest — because Haskell is pure:
* you can see a function as "some data with holes in it" rather than a machine routine
* function definitions are literally a type of definitional equality

So if:

```haskell
display Foo = "my awesome message"
```

...then...

```haskell
display Foo == "my awesome message"
```

in 100% of cases. It's a simple case, but the same goes for all of your functions, even ones with `throw` and whatnot, because underneath that's all modeled in data (just hidden by a monadic interface to make it feel more imperative)